calico-enterprise/reference/installation/helm.mdx

```diff
@@ -0,0 +1,137 @@
+---
+description: Configure Calico Enterprise resources with helm installation
+---
+
+## Helm Installation Reference
+
+You can customize your {{prodname}} installation by providing settings via `values.yaml` during helm-based installation.
+Your custom configurations for any of the properties for the resources listed below can be set in your custom
+`values.yaml` file.
+
+- [Installation](../reference/installation/api#operator.tigera.io/v1.InstallationSpec)
+- [Api server](../reference/installation/api#operator.tigera.io/v1.APIServerSpec)
+- [Compliance](../reference/installation/api#operator.tigera.io/v1.ComplianceSpec)
+- [Intrusion detection](../reference/installation/api#operator.tigera.io/v1.IntrusionDetectionSpec)
+- [Log collector](../reference/installation/api#operator.tigera.io/v1.LogCollectorSpec)
+- [Log storage](../reference/installation/api#operator.tigera.io/v1.LogStorageSpec)
+- [Manager](../reference/installation/api#operator.tigera.io/v1.ManagerSpec)
+- [Monitor](../reference/installation/api#operator.tigera.io/v1.MonitorSpec)
+- [Policy recommendation](../reference/installation/api#operator.tigera.io/v1.PolicyRecommendationSpec)
+- [Authentication](../reference/installation/api#operator.tigera.io/v1.AuthenticationSpec)
+- [Application layer](../reference/installation/api#operator.tigera.io/v1.ApplicationLayerSpec)
+- [Amazon cloud integration](../reference/installation/api#operator.tigera.io/v1.AmazonCloudIntegrationSpec)
+- [Default felix configuration](../reference/resources/felixconfig#spec)
+
+Any of the configs available for any of the resources above can be configured via helm installation. For a list of all
+available properties for each resource please refer to the [Installation reference](../reference/installation/api) page.
+
+You still can change / apply settings after installation to make changes to your cluster by applying any custom resource.
+
+Note:
```

Comment:
:::note
If you customize resources at time of installing {{prodname}}, the `v1` apiVersion is used. However, when you customize resources after installation (when the `tigera-apiserver` is running), use the `v3` apiVersion.